tests/console/zypper_lifecycle.pm

```diff
@@ -52,7 +51,16 @@ sub run {
     if ($output =~ /name="(?<package>[^"]+)"/) {
         $package = $+{package};
     }
-
+    # For JeOS build testing we are always using the latest repositories, because
+    # JeOS images are build with a *different* build number to SLES/Leap. It seems
+    # that SLES repositories are not populated with packages for several hours
+    # (I have seen the test pass after 14 hours from the image creation), and actually
+    # no package in the image comes from any repository - it's from the image. So we
+    # hard-code 'sles-release' package, and it... works. Somehow.
+    if (!$package && is_jeos) {
+        record_soft_failure "Hardcoding 'sles-release' package for lifecycle check";
+        $package = 'sles-release';
```

Comment:
Can this one be resolved with proper sync of repos for JeOS? As what we do for SLES is that we sync repos first and only then trigger jobs. And as I've introduced some changes to parsing recently, was this behavior same 3-4 weeks ago? As this is pretty cheap solution, we still can merge it, but I wonder if we can address it in a better way.

Reply:
Michal is on vacation this week so do not expect fast answer.

Reply:
@mnowaksuse any updates on this one?

Reply:
Great questions, I wish I had great answers for you :).

I can't think of any sync problems because for JeOS the SLES repository where 'sles-release' package is located, always exist due to the universal '0000' build number proxy, which always points to the latest SLES repository.

This behavior was here from the beginning of time, no change.

Reply:
Ok, let's try it out. But I still think that we can skip triggering the build before repos are consistent and parse correct build numbers. So, @mnowaksuse if you'll get an idea what has to be changed - let me know.

Reply:
Thanks @rwx788 :). And be it you I fix a regression on top of it! :) See: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4111.